app/views/admin/worldwide_organisations/_form.html.erb

```diff
@@ -62,6 +62,7 @@
     rows: 4,
   } %>
 
+  <%= hidden_field_tag "worldwide_organisation[default_news_image_attributes][id]", worldwide_organisation.default_news_image.id %>
```

Comment:
a little bit more railsy would be to `<%= form.fields_for :default_news_image do |_image_fields| %>`